doc/quickstart/install.rst

```diff
@@ -208,3 +208,20 @@ yourself if you are just running diagnostics:
  - DKRZ-Mistral: `esmvaltool` is available on login nodes (`mistral.dkrz.de`) and pre- and post-processing
    nodes (`mistralpp.dkrz.de`) after login and module loading via `module load esmvaltool`; the command
    `module help esmvaltool` provides some information about the module.
+
+Installation from the conda lock file
```

Comment:
Very useful to have docs with this new feature 🎉

Some comments to improve this section in the docs:
- in rst, code needs two backtics (``); a single one renders as cursive text. (Seems to have gone awry in the bit of text just above as well?)
- Some line breaks would be helpful too! Especially commands may be best placed inside code blocks:
  ```
   .. code-block:: bash
   ```
- Perhaps add the command to install conda-lock:
   ```
   conda install -y conda-lock -c conda-forge
   ```

Reply:
cheers, will do!